### PR TITLE
Update UG to be consistant with v1.3

### LINKED ISSUE #95: Update UG to be consistant with v1.3

### PR DESCRIPTION
**Pull request by kinjalagarwal1810**
closes #95

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -81,10 +70,61 @@ Throughout the guide, you'll encounter various icons that signify different type
...
+  * Requires members to accumulate a specified threshold of membership points. 
+  * Unlocks enhanced benefits and rewards.
+  
+* Gold Tier:
+  * A higher tier requiring more membership points. 
+  * Offers substantial rewards and exclusive offers. 
+
+* Platinum Tier:
+  * The pinnacle of the membership hierarchy. 
+  * Members enjoy the best rewards and personalized services.
```
Comment (billyhoce) [inline docs/UserGuide.md:115]: I think can just put a table with the tier name and points required. Maybe also want to put a tip that the name, number, and point requirement of Tiers cannot be changed for now, and is still in development. Users should adjust the number of points their items give in order to suit their use

File: docs/UserGuide.md [hunk 2]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
-Furthermore, certain edits can cause the AddressBook to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
-</div>
+SweetRewards data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-### Archiving data files `[coming in v2.0]`
+⚠️ **Warning**: If your changes to the data file makes its format invalid, SweetRewards will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
+Furthermore, certain edits can cause SweetRewards to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 
+## 4.18 Archiving data files 
+`[coming in v2.0]`
```
Comment (billyhoce) [inline docs/UserGuide.md:398]: I think can just remove this?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
+| Membership Tiers  | Designated levels within the loyalty program offering different benefits, determined by the amount of points a member has accumulated.  |
+| Seed Data         | Preloaded data used to demonstrate the application's capabilities without needing to enter new data manually.                           |
+| Allergen          | Allergens that customers have, for the baker to take note when making an order.                                                         |
+| GF                | Gluten-free: An allergen specification.                                                                                                 |
+| LF                | Lactose-free: An allergen specification.                                                                                                |
+| Order Details     | Information related to purchases made by a member, used for tracking the orders made by the customer.                                   |
+| Points            | A system of credit awarded to members for their purchases, contributing to their membership tier placement.                             |
```
Comment (billyhoce) [inline docs/UserGuide.md:416]: Not sure if Order Details and Points are needed here?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -3,65 +3,54 @@ layout: page
...
-* **Update membership tiers**: Elevate members' tiers based on their accumulated points, unlocking new rewards and benefits aligned with each tier.
-
-### Understanding Membership Tiers
-One of the core features of SweetRewards is the Membership Tier System. This system rewards your loyal customers by **categorizing them into different tiers** based on the points they have accumulated through purchases at your bakery. Each tier offers different benefits, such as discounts, free items, or exclusive products that can be customised by you. Here is a breakdown of the tiers:
-
-**Bronze Tier**: This is the starting tier for all new members. Members need to accumulate at least 100 points to move to the next tier.
-**Silver Tier**: Members who have accumulated 200 points are upgraded to the Silver Tier, which offers more benefits than the Bronze Tier.
-**Gold Tier**: Members who have accumulated 500 points are upgraded to the Gold Tier, which offers more benefits than the Silver Tier.
-**Platinum Tier**: This is the highest tier, reserved for members who have collected 1000 points. Gold Tier members receive the best benefits, underlining their importance to your bakery.
```
Comment (kab-dot) [inline docs/UserGuide.md:56]: should be Platinum tier members receive....

**Review by kab-dot — COMMENTED** [review]
Other than a few errors, and fixing of the addmship, LGTM!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install SweetRewards on the other computer and copy the data file from your original installation directory to the new one. This will replace the empty data file created during the new installation.
+
+**Q**: How do I manually add points to a member's account?<br>
+**A**: Use the addpoints command with the member's name and the desired amount of points. For instance: addpoints n/John Doe p/50 adds 50 points to John Doe's account, useful for promotional points or correcting errors.
 
-**Q**: How can I add points to a member's account?
-**A**: Use the addpoints command with the member's name and the number of points you want to add. For example, addpoints n/John Doe p/50 adds 50 points to John Doe’s account.
+**Q**: How can I upgrade a member's tier manually?<br>
+**A**: Employ the addmship command with the member's name and the intended new tier. For example, addmship n/John Doe ms/Gold manually upgrades John Doe to the Gold tier, ideal for special recognitions.
```
Comment (billyhoce) [inline docs/UserGuide.md:430]: I think this is outdated, only can add points so:

Employ the addmempts command with the member's name and the points you want to add. For example, addmempts n/John Doe mp/400 manually awards John Doe 400 points. The membership tier of the member will be determined by the number of points they have according to the **membership tier table** <----- (Add a hyperlink to table here)

File: docs/UserGuide.md [hunk 6]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
+**Q**: Is it possible to remove a member from the loyalty program?<br>
+**A**: Yes, execute the delete command with the index number of the member in the displayed list. E.g., delete 3 removes the third member in the list, used for cleaning up duplicate entries or removing inactive accounts.
 
-**Q**: Can I delete a member from the loyalty program?
-**A**: Yes, you can use the delete command followed by the member's index number in the list. For example, delete 3 will delete the third member listed in the current view.
+**Q**: How do membership points influence tier levels?<br>
+**A**: Membership tiers are determined by the total accumulated membership points. Each tier requires reaching a specific points threshold, which is automatically tracked by the system. Check the "Understanding Membership Tiers and Points System" section for detailed tier requirements.
+
+**Q**: How do I handle points for cancelled orders?
+**A**: For cancelled, use the `redeempoints` command to deduct the equivalent points from the member's account, keeping their points balance accurate with their actual purchases.
```
Comment (billyhoce) [inline docs/UserGuide.md:439]: Good qn and answer!

File: docs/UserGuide.md [hunk 7]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
-### Technical Issues
-1. Application doesn't start
-   * 🚫 Problem: Double-clicking the jar file doesn’t open SweetRewards. 
-   * ✅ Solution: Ensure Java 11 or above is installed on your machine. Open a command terminal, navigate to the folder containing the jar file, and run java -jar sweetrewards-v1.2.jar. If the problem persists, verify that the downloaded jar file is not corrupted and re-download if necessary.
+## Technical Issues
+1. Application won't start<br>
+🚫 **Problem**: Double-clicking the jar file doesn’t open SweetRewards.<br> 
+✅ **Solution**: Verify that Java 11 or higher is installed. Open a command prompt, navigate to the application's directory, and execute java -jar sweetrewards-v1.2.jar. If issues persist, re-download the jar file to rule out corruption.
+
+
```
Comment (billyhoce) [inline docs/UserGuide.md:452]: Blank line delete

File: docs/UserGuide.md [hunk 8]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
-   * 🚫 Problem: When using multiple screens, SweetRewards opens off the visible screen. 
-   * ✅ Solution: Delete the preferences.json file found in the same folder as the jar file. Restart SweetRewards.
 
-### Operational Questions
-1. Adding Points to Wrong Member
-    * 🚫 Problem: Points were mistakenly added to the wrong member. 
-    * ✅ Solution: Use the edit command to adjust the points for both involved members. Replace the newly updated (wrong) number of points with the initial (correct) number of points and add the points for the new member again. For example, edit 2 p/50 to replace the wrongly updated points and addpoints 3 p/50 to add points to the correct member.
+3. Orders Not Reflecting in Member's History<br>
+🚫 **Problem**:  Newly added orders are not reflected in a member's history.<br>
+✅ **Solution**: Verify that `addorder` was used correctly, including the correct member name and order details. This ensures all transactions are accurately recorded.<br>
```
Comment (billyhoce) [inline docs/UserGuide.md:470]: Oh! Good catch here. Solution should be click off the person can click back. Ill edit this!

File: docs/UserGuide.md [hunk 9]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
+| **[Edit](#411-editing-a-member--edit)**                                               | `edit INDEX [n/NAME] [hp/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [ag/ALLERGEN]…​`          | `edit 2 n/James Lee e/jameslee@example.com`                                                   |     
+| **[Find](#412-locating-members-by-name--find)**                                       | `find KEYWORD [MORE_KEYWORDS]`                                                         | `find James Jake`                                                                             |
+| **[List](#44-listing-all-members--list)**                                             | `list`                                                                                 | `list`                                                                                        |
+| **[Help](#41-viewing-help--help)**                                                    | `help`                                                                                 | `help`                                                                                        |
```
Comment (billyhoce) [inline docs/UserGuide.md:492]: Help command has a new `--tag`, might need to add here

File: docs/UserGuide.md [hunk 10]
```diff
@@ -303,110 +374,121 @@ Format: `clear`
...
+| **[Add Points](#48-adding-points--addpoints)**                                        | `addpoints n/MEMBER_NAME p/POINTS`                                                     | `addpoints n/John Doe p/50`                                                                   |
+| **[Redeem Points](#49-redeeming-points--redeempoints)**                               | `redeempoints n/MEMBER_NAME p/POINTS`                                                  | `redeempoints n/John Doe p/50`                                                                |
+| **[Update Membership](#410-updating-membership--addmship)**                           | `addmship n/MEMBER_NAME ms/MEMBERSHIP_TIER`                                            | `addmship n/John Doe ms/T2`                                                                   |
+| **[Clear](#414-clearing-all-entries--clear)**                                         | `clear`                                                                                | `clear`                                                                                       |
```
Comment (billyhoce) [inline docs/UserGuide.md:487]: Maybe should include `--force` here

**Review by billyhoce — COMMENTED** [review]
Overall, very good changes! Good work!

**Pull request merged**